src/components/transformation/create/InitializeDraftButton.tsx

```diff
@@ -74,9 +79,28 @@ function InitializeDraftButton({
 
                 setSourceCollections(collections);
 
+                let latestTransformVersions: { [tableName: string]: number } =
+                    {};
+
                 const transformConfigs: TransformConfig[] = collections.map(
-                    (collection) =>
-                        templateTransformConfig(collection, entityName)
+                    (collection) => {
+                        const tableName = stripPathing(collection);
+
+                        if (Object.hasOwn(latestTransformVersions, tableName)) {
+                            latestTransformVersions[tableName] += 1;
+                        } else {
+                            latestTransformVersions = {
+                                ...latestTransformVersions,
+                                [tableName]: 0,
+                            };
+                        }
```

Comment:
What is driving this approach? Is something wrong with just using the block notation for both parts of the if/else block? Then the object could become a `const`

```javascript
if (Object.hasOwn(latestTransformVersions, tableName)) {
    latestTransformVersions[tableName] += 1;
} else {
    latestTransformVersions[tableName] = 0;
}
```

Reply:
The answer to a related comment applies here:

> The answer: typescript code presets on assistive technologies. Hence, I perform an exhaustive review of my own code changes before merging the PR to catch any superfluous syntax introduced by the assistive technology.